src/lib/components/tablev2/SearchWithQueryParams.tsx

```diff
@@ -18,7 +17,7 @@ export function SearchWithQueryParams(props) {
   function handleOnChange(value) {
     const { onChange } = props;
     params.set(queryParams, value);
-    history.replace(`${match.url}?${params.toString()}`);
+    history.replace(`${pathname}?${params.toString()}`);
```

Comment:
All looks good, but I'm not familiar with the situation - do we want to add new history entry (thus enabling going back using the browser's back button) or replace it?